pkg/measurement/measure.go

```diff
@@ -7,10 +7,13 @@ import (
 	"time"
 )
 
+const DefaultMaxLatency = 16 * time.Second
```

Comment:
16s is too small for TPC-H and CH-benCHmark, they usually take one minute to ten minutes, and more `sf` means a longer duration

Reply:
Seems an unreasonable report on TPC-H:
```
[Summary] Q1: 10.33s
[Summary] Q10: 15.84s
[Summary] Q11: 14.23s
[Summary] Q12: 15.19s
[Summary] Q13: 10.44s
[Summary] Q14: 14.87s
[Summary] Q15: 15.57s
[Summary] Q16: 12.62s
[Summary] Q17: 14.50s
[Summary] Q18: 14.90s
[Summary] Q19: 10.33s
[Summary] Q2: 11.51s
[Summary] Q20: 12.48s
[Summary] Q21: 15.84s
[Summary] Q22: 10.80s
[Summary] Q3: 15.84s
[Summary] Q4: 15.84s
[Summary] Q6: 7.11s
[Summary] Q7: 15.84s
[Summary] Q8: 15.84s
[Summary] Q9: 14.76s
```